DataFormats/METReco/interface/HcalPhase1FlagLabels.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DataFormats_METReco_HcalPhase1FlagLabels_h
+#define DataFormats_METReco_HcalPhase1FlagLabels_h
+
+#include "DataFormats/METReco/interface/HcalCaloFlagLabels.h"
+
+namespace HcalPhase1FlagLabels
+{
+    enum HBHEStatusFlag {
+        /*  0 */ HBHEHpdHitMultiplicity = HcalCaloFlagLabels::HBHEHpdHitMultiplicity,
+        /* 11 */ HBHEIsolatedNoise = HcalCaloFlagLabels::HBHEIsolatedNoise,
+        /* 12 */ HBHEFlatNoise = HcalCaloFlagLabels::HBHEFlatNoise,
+        /* 13 */ HBHESpikeNoise = HcalCaloFlagLabels::HBHESpikeNoise,
+        /* 15 */ HBHETS4TS5Noise = HcalCaloFlagLabels::HBHETS4TS5Noise,
```

Comment:
why not simply edit HcalCaloFlagLabels?
partial reuse of the same values looks odd.

IMO, These flags are too high level to pass downstream support of different meanings for phase1 vs not phase1.
We will need to support re-reco of "legacy" HF. 
It seems simpler to preserve common phase space for flag enums.